engine/src/main/scala/cromwell/engine/workflow/lifecycle/execution/WorkflowExecutionActor.scala

```diff
@@ -557,14 +572,14 @@ final case class WorkflowExecutionActor(workflowId: WorkflowId,
         factories.get(backendName) match {
           case Some(factory) =>
             val ejeActorName = s"${workflowDescriptor.id}-EngineJobExecutionActor-${jobKey.tag}"
-            val ejeActor = context.actorOf(EngineJobExecutionActor.props(data, factory, initializationData.get(backendName), restarting), ejeActorName)
+            val ejeActor = context.actorOf(EngineJobExecutionActor.props(jobKey, data, factory, initializationData.get(backendName), restarting), ejeActorName)
             pushNewJobMetadata(jobKey, backendName)
 
-            ejeActor ! EngineJobExecutionActor.Execute(jobKey)
+            ejeActor ! EngineJobExecutionActor.Execute
 
             Success(WorkflowExecutionDiff(Map(jobKey -> ExecutionStatus.Starting)))
           case None =>
-            throw new WorkflowExecutionException(List(new Exception(s"Could not get BackendLifecycleActor for backend $backendName")))
+            throw WorkflowExecutionException(List(new Exception(s"Could not get BackendLifecycleActor for backend $backendName")))
```

Comment:
TOL: Just for usability, there could be an alt constructor which doesn't take a `List` - this does seem slightly clunky